### ISSUE TITLE
Review and set up analytics

### ISSUE THREAD

**Issue by Benbaillou** [hackforla/food-oasis#1127]
### Overview
Work on Google Analytics usability to understand Foodseeker traffic in order to optimize the user funnel from acquisition to engagement

Here is a quick recap of FOLA map interface criteria in order to understand the right metrics to set up in google analytics
 
**What are you building?**

**What user problem are we solving? How? Why did we choose this solution?

**What goal is this addressing? How will we measure success?**

North Star Metric
Number of clicked directions or call or email

Success metric
Acquisition: user origin, how many per week
Activity: 
  what and how many paged viewed, 
  nb clicked on Pantry or meal buttons, 
  nb clicked on pings or panel - 
  time of visits
  what was seach in the tool bar
  geography of the users and their searches

Counter metric
nb of visits without clicked directions or call or email

 **What are the key unknowns or risks of this project? How are we mitigating them?**

**Product scope**
User flow - 

Wireframe 

### Action Items

- [ ] Audit the existing tags and displayed data and optimize them if needed 
- [ ] Define a user flow on the website (wireframes)
- [ ] Set up Metrics and expectations
- [ ] Determine point of improvement backed with UX researches

NEW HOTJAR ACTION ITEMS
- [ ]  Check if we have access to the hotjar analytics results ( @entrotech might know more )
- [ ]  Find out the history about if it was useful or not
- [ ]  Decide if we want to use hotjar or not, and if not, remove it from our site.

### Resources/Instructions
Google Analytics account

**Comment by Benbaillou** [comment]
note for myself: how to analyse search bar types

**Comment by Benbaillou** [comment]
### Note to myself

**Estimated market**
According to this report: 1,2 million of Angelinos experience food insecurity and they look for assistance 9 times per year
This gives 400,000 people looking for food assistance per week
[HungerFacts-Report.pdf](https://github.com/hackforla/food-oasis/files/7686813/HungerFacts-Report.pdf)
Our current number of visits per week is 40 

**GoogleAnalyics question**
- acquisition: who are our 40 users pw ?
- event tags: get a description/knowledge of what each event means


**To do**
- Draw an acquisition and conversation funnel
-  do a test of paid social (FB and Instagram)
- define what a conversation looks like

**Comment by fancyham** [comment]
Some actions we can take that we discussed today:

- [ ] Find out what analytics we currently have (might review source code of site — I saw only Firebase Analytics (AKA Google Analytics)
- [ ] Get everyone interested access to those analytics
- [ ] Discuss among team what stats are useful for us
- [ ] Identify gap between what we have currently and what we want
- [ ] Implement what we can

**Comment by fancyham** [comment]
Would like to see what is being put into the search field — are people adding organization names?  I'd try that if I knew about a pantry.

- #1115 
- #1112

**Comment by staceyrebekahscott** [comment]
@itserindean FYI, I am marking this as a priority of the Impact Sprints and have added the proper label, and marked the milestone as Project Organization (this might change). Note that I also moved #1144 , which was in Icebox and is related to this issue, as per the Tracked In marker at the top, to the Prioritized Backlog. Just wanted to bring that to your attention as you start work on this issue, that the 2 issues are connected.

**Comment by staceyrebekahscott** [comment]
Moving this to In Progress, as per the process of only having people assigned to issues they are actively working on.

**Comment by staceyrebekahscott** [comment]
@itserindean Upon reviewing this issue more closely, wasn't some of this addressed in the Heuristics evaluation? 

It seems like the scope of this issue is broad and needs a bit of refining to clarify, and possibly broken down into multiple issues.

**Comment by itserindean** [comment]
@staceyrebekahscott no, I don't think any of this was addressed by heuristics. 
@entrotech has sent me instructions for accessing the Google analytics for our site and I have now been added to the correct 1password vault (devOPs) so I can set it up on my end. I'll have more of an update at tomorrow's PM meeting.

**Comment by staceyrebekahscott** [comment]
@itserindean OK, Maybe I am misunderstanding one of the Action Items listed above, re:  Determine point of improvement backed with UX researches, or the overview in general is confusing to me. 

I look forward to your update, and a clearer explanation of what we can learn from the analytics.

**Comment by staceyrebekahscott** [comment]
@itserindean What else are you intending to do on this issue? Can it be closed now that you have the analytics set up?

**Comment by itserindean** [comment]
@staceyrebekahscott None of the action items on this have been accomplished yet and to be honest I need todo some learning to even know where to start

**Comment by staceyrebekahscott** [comment]
@itserindean Given that this issue was written in Nov 2021 by a PM who is no longer with the project, we can use some discretion as to how much we want to adhere to what was intended when this issue was originated. It is possible that these results might not tell us much at this time. And now that we are in the early stages of roadmapping some Food Seeker initiatives, perhaps it is more important right now that the analytics are in place, and the interpretations of the analytics might be more useful in the future. 

I believe your efforts might be better spent in other areas at this time. I would recommend closing this issue, and then when we have completed some changes to the site that need interpretation and feedback we can start the analytics review process with a fresh scope based on current needs. 

Please proceed based on your judgement, the above is just my opinion.

**Comment by fancyham** [comment]
I think the first step would be to

- [ ] 1. Determine what we'd like to get information about (and know why) {this would need to be added to the top description}
- [ ] 2. See what we have set up currently and can we derive answers to the above questions?
- [ ] 3. Get a delta of the two above
- [ ] 4. Figure out how to fill in the gaps — it might be site analytics, it might be user research talking to individuals, orgs, food-seekers, or digesting existing research

I would suggest for the first one, a discussion about what our high and low-priority goals are (e.g. helping the most people possible, native languages of food seekers) and identifying how we'd know if we're doing that via what metrics. (e.g. how many people view a single org detail page, how many people view multiple org detail pages, where are people visiting from?).

Ideally the above in concert with the research team.

Also figuring out what things we can't answer with site analytics and then figuring out a plan with UX research team.

If we get into this, perhaps I can help with @GigiUxR to discuss some things that are doable with our current resources.

**Comment by fancyham** [comment]
Also, I think there was a PM (Ryland?) who knew their way around Google Analytics really well?  I've done some work with it, so I can also help a bit.

We should figure out what's doable and most useful rather than get overwhelmed.  Analytics is a steep learning curve but is a valuable skill, so we'll want to bring in help/experts/people who are really into learning/doing it. :)

**Comment by staceyrebekahscott** [comment]
I agree with the scope presented by @fancyham. I think my point earlier was more that @itserindean shouldn't get too bogged down with this learning curve. If one of the new researchers is familiar with Google Analytics this might be a good issue for them to tackle, once the metrics are established. Erin- would you be OK with taking lead on establishing the metrics, with PM support? Then it would be ready for delegation to a researcher.

**Comment by itserindean** [comment]
Closing since this is now covered by #1350

**Comment by fancyham** [comment]
Hey, I noticed during #928 that [we have a hotjar.com analytics package](https://github.com/hackforla/food-oasis/issues/928#issuecomment-1266216179) running on our site that is tracking what our visitors do, but I think we don't have access to those results?

TL;DR: If we're not using hotjar and don't intend to, then we're wasting bandwidth for our users and exposing them to unnecessary tracking and we should remove it.

So @itserindean, not sure if this issue is the correct place for this but I'm suggesting re-opening or creating a new issue to…
- [ ] Check if we have access to the hotjar analytics results ( @entrotech might know more )
- [ ] Find out the history about if it was useful or not
- [ ] Decide if we want to use hotjar or not, and if not, remove it from our site.

Details:

Hotjar's analytics (heatmaps and what people clicked on) could be interesting but the history, as I recall, was that a PM in early 2021 was really interested in it, got it installed as a trial version, but I don't think we ever got access to the account after the PM rolled off the project.

I think that there were also problems getting it to work well on our [SPA (single page app)](https://help.hotjar.com/hc/en-us/articles/115011805428-Hotjar-on-Single-Page-Apps) website.

**Comment by itserindean** [comment]
reopening to answer questions re: hotjar

**Comment by itserindean** [comment]
at the dev meeting @entrotech explained that we had set up hotjar but in a trial capacity that has long expired.
We have not been able to get access to the analytics.
